auslib/web/common/rules.py

```diff
@@ -20,6 +21,25 @@ def get_rules():
             where[field] = request.args[field]
 
     rules = dbo.rules.getOrderedRules(where=where)
+
+    names = set(chain.from_iterable((rule['mapping'], rule['fallbackMapping']) for rule in rules))
+    blobs = dbo.releases.getReleaseBlobs(names=names)
+
+    for rule in rules:
+        if rule['mapping']:
+            b = blobs.get(rule['mapping'], {})
+            rule['mapping'] = {
```

Comment:
It looks like something weird is happening in the UI because of this change; when I try to edit a Rule I end up with [object Object] for the Mapping and Fallback Mappinig values: https://screenshots.firefox.com/42aqbbyNjv9CN0fc/localhost.

It might be easier if you split this into two properties: has_wnp_mapping, has_wnp_fallbackMapping. I'm open to other solutions, too.

Reply:
I'm open to trying it, but please do a lot of testing -- there's a lot of things in the frontend that assume the rule state can be read and written to. In particular, please make sure that creating/updating/delete rules and rule scheduled changes still continues to work, and that the UI displays changes to them correctly with and without a refresh.

Reply:
Will do. Thanks!